pytracking/tracker/base/basetracker.py

```diff
@@ -52,6 +52,60 @@ def track_sequence(self, sequence):
 
         return tracked_bb, times
 
+    def track_videofile(self,videofilepath):
+        "Run track with a video file input."
+
+        #cap = cv.VideoCapture("/home/dtt/Videos/haze/v_1509066461577.avi")
+        cap = cv.VideoCapture(videofilepath)
+        display_name = 'Display: ' + self.params.tracker_name
+        cv.namedWindow(display_name, cv.WINDOW_NORMAL | cv.WINDOW_KEEPRATIO)
+        cv.resizeWindow(display_name, 960, 720)
+
+        while True:
+            success, frame = cap.read()
+            if success is not True:
+                break
+            cv.imshow(display_name, frame)
+            k = cv.waitKey(100)
+            if k == 27:
+                x, y, w, h = cv.selectROI(display_name, frame)
```

Comment:
would be nice to set ```fromCenter=False``` in selectROI so that the top left and bottom right points are selected, same as in run_webcam function